services/ux-backend/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+
+	handler "github.com/red-hat-storage/ocs-operator/v4/services/ux-backend/handlers"
+	"k8s.io/klog/v2"
+)
+
+var tokenLifetimeInHours int
+var listenPort int
+
+func init() {
+
+	var err error
+	tokenLifetimeInHours, err = strconv.Atoi(os.Getenv("ONBOARDING_TOKEN_LIFETIME"))
+	if err != nil || tokenLifetimeInHours <= 0 {
+		tokenLifetimeInHours = 48
+	}
+	klog.Info("generated token will be valid for ", tokenLifetimeInHours, " Hours")
```

Comment:
1. To control the way we present the values use the formatted version of Info -> Infof
```suggestion
	klog.Infof("generated token will be valid for %d hours", tokenLifetimeInHours)
```

2. This message is not very helpful. You do not capture the source of the value (default or user-provided). You do not exit in cases where the user provided a value but it was malformed. Please  take a look at a previous comment about UX_BACKEND_PORT env var for reference

Reply:
Done